build.gradle

```diff
@@ -132,6 +132,7 @@ dependencies {
     compile 'org.fxmisc.flowless:flowless:0.6'
     compile 'org.fxmisc.richtext:richtextfx:0.8.2'
     compile 'com.sibvisions.external.jvxfx:dndtabpane:0.1'
+    compile 'javax.inject:javax.inject:1'
```

Comment:
Is that an official java dep?

Reply:
Yes, it's a popular JavaEE dependency, see https://mvnrepository.com/artifact/javax.inject/javax.inject.